appengine/flexible/scipy/requirements.txt

```
@@ -5,5 +5,5 @@ numpy==2.0.0; python_version > '3.9'
 numpy==1.26.4; python_version == '3.9'
 numpy==1.24.4; python_version == '3.8'
 pillow==10.3.0
-scipy==1.10.1; python_version <= '3.9'
-scipy==1.14.0; python_version > '3.9'
+scipy==1.14.1; python_version <= '3.9'
```

SciPy 1.12.0 dropped support, Renovate forced bump ignoring python_version. Thus should be the max 3.8-supported version. 
```suggestion
scipy==1.11.4; python_version <= '3.9'
```